File: tests/adapter-fixtures/java/Basic.java
// SPDX-License-Identifier: BUSL-1.1
// Copyright 2026 Firelock, LLC

// Fixture: basic Java entities
// Expected: 2 methods, 1 class

public class Basic {
    private String name;

    public Basic(String name) {
        this.name = name;
    }

    public String getName() {
        return this.name;
    }

    private int helperInternal() {
        return 42;
    }
}

File: tests/adapter-fixtures/go/basic.go
// SPDX-License-Identifier: BUSL-1.1
// Copyright 2026 Firelock, LLC

// Fixture: basic Go entities
// Expected: 2 functions, 1 struct

package user

type UserProfile struct {
	Name  string
	Email string
}

func CreateUser(name, email string) UserProfile {
	return UserProfile{Name: name, Email: email}
}

func helperInternal() int {
	return 42
}
